test/e2e/e2e.go

```diff
@@ -34,6 +34,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	runtimeutils "k8s.io/apimachinery/pkg/util/runtime"
 	clientset "k8s.io/client-go/kubernetes"
+	federationtest "k8s.io/kubernetes/federation/test/e2e"
```

Comment:
Do you plan to remove this in the future?

Reply:
yes, we plan to remove this from k/k and we shall do it as soon as we build the ci jobs on k/f and testing works from there.